src/main/java/tutoraid/logic/parser/AddStudentToLessonCommandParser.java

```diff
@@ -15,8 +17,8 @@
 public class AddStudentToLessonCommandParser implements Parser<AddStudentToLessonCommand> {
 
     /**
-     * Parses the given {@code String} of arguments in the context of AddStudentToLessonCommand
-     * and returns an AddStudentToLessonCommand object for execution.
+     * Parses the given {@code String} of arguments in the context of AddStudentsToLessonsCommand
+     * and returns an AddStudentsToLessonsCommand object for execution.
```

Comment:
`AddStudentsToLessonsCommand` should be `AddStudentToLessonCommand` instead, as seen from the return type